conf/messages.en

```diff
@@ -33,6 +33,10 @@ date.month = Month
 date.year = Year
 date.empty.error = Error: {0} cannot be empty
 
+error.title = Sorry, there is a problem with the service
+error.detail1 = Try again later.
+error.detail2 = We saved your answers. They will be available for 30 days.
```

Comment:
"We saved your answers". This is not true in the case of enrolment failure, is it?

Reply:
We have introduced a new error page which does not mentioned the saving of answers.